src/Type/Definition/ObjectType.php

```diff
@@ -148,10 +147,14 @@ public function getInterfaces(): array
                 );
             }
 
-            /** @var InterfaceType[] $interfaces */
-            $interfaces = array_map([Schema::class, 'resolveType'], $interfaces ?? []);
-
-            $this->interfaces = $interfaces;
+            $this->interfaces = [];
+            if (is_array($interfaces)) {
+                foreach ($interfaces as $interface) {
+                    /** @var InterfaceType $interfaceType Ensured to be true during schema validation */
+                    $interfaceType      = Schema::resolveType($interface);
+                    $this->interfaces[] = $interfaceType;
+                }
+            }
```

Comment:
```suggestion
            foreach ($interfaces ?? [] as $interface) {
                /** @var InterfaceType $interfaceType Ensured to be true during schema validation */
                $interfaceType      = Schema::resolveType($interface);
                $this->interfaces[] = $interfaceType;
            }
```

also seems more readable without any cost